README.md

```diff
@@ -41,7 +41,9 @@ To further configure your plugin, use the following custom parameters in your `s
 | `enableASM`                   | Enable [Datadog Application Security Management (ASM)][19] on the Lambda function. Requires the Datadog extension to be present (using `addExtension` or manually added) and `enableDDTracing`. Defaults to `false`.                                                                                                                                                                                                                                                                                                                                                          |
 | `enableDDLogs`                | Enable Datadog log collection using the Lambda Extension. Defaults to `true`. Note: This setting has no effect on logs sent by the Datadog Forwarder.                                                                                                                                                                                                                                                                        |
 | `monitors`                    | When defined, the Datadog plugin configures monitors for the deployed function. Requires setting `DATADOG_API_KEY` and `DATADOG_APP_KEY` in your environment. To learn how to define monitors, see [To Enable and Configure a Recommended Serverless Monitor](#to-enable-and-configure-a-recommended-serverless-monitor).                                                                                                    |
-| `captureLambdaPayload`        | [Captures incoming and outgoing AWS Lambda payloads][17] in the Datadog APM spans for Lambda invocations. Defaults to `false`.                                                                                                                                                                                                                                                                                               |
+| `captureLambdaPayload`        | [Captures incoming and outgoing AWS Lambda payloads][17] in the Datadog APM spans for Lambda invocations. Defaults to `false`.     
+| `captureCloudRequestPayload`        | [Captures request data exchanged between your application and AWS services][20] in the Datadog APM spans. Values: `all`. To [protect sensitive data][21], use JSON Path syntax.. Defaults to `false`.           
```

Comment:
```suggestion
| `captureCloudRequestPayload`        | When using an AWS SDK client within your lambda function, captures [request data exchanged between it and AWS services][20] in the Datadog APM spans. Values: `all`. To [protect sensitive data][21], use JSON Path syntax. Defaults to `""`.           |
```
Make sure the whitespace allows that last `|` to line up correctly, I can't do it very well from the UI.